lab-matt/data/model/969ee620-23a9-11e6-8012-5d64436d5034

```diff
@@ -0,0 +1 @@
+{"id":"969ee620-23a9-11e6-8012-5d64436d5034","name":"Captain Underpants","luckNumber":5,"timestamp":"2016-05-27T01:23:16.482Z"}
```

Comment:
In the future gitignore your data folder, anything you program needs should be created at runtime (like a data folder).